pkg/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.TensorFlow.Redist.nupkgproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>$(MSBuildProjectName) contains the TensorFlow C library version $(TensorFlowVersion) redistributed as a NuGet package.</PackageDescription>
+    <PackageLicenseUrl>https://github.com/tensorflow/tensorflow/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://www.tensorflow.org/install/install_c</PackageProjectUrl>
+    <PackageReleaseNotes>https://github.com/tensorflow/tensorflow/releases/tag/v$(TensorFlowVersion)</PackageReleaseNotes>
+    <PackageTags>$(PackageTags) TensorFlow</PackageTags>
+    <!-- TODO: consider PackageIconUrl -->
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <Content Include="..\common\CommonPackage.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" />
```

Comment:
Question (probably for a lawyer):  Do we need to put a license/notice in this package?  or is using the `PackageLicenseUrl` enough to meet TF requirements?

Reply:
We have a discussion going around this and will find out all the details.

Reply:
We have a discussion going around this and will find out all the details.

---
In reply to: [212419037](https://github.com/dotnet/machinelearning/pull/720#discussion_r212419037) [](ancestors = 212419037)

Reply:
TF included a license in their zips so I went ahead and placed it in the root of this package.

---
In reply to: [212426813](https://github.com/dotnet/machinelearning/pull/720#discussion_r212426813) [](ancestors = 212426813,212419037)